projects/common/de10nano/de10nano_system_assign.tcl

```diff
@@ -0,0 +1,704 @@
+# de10nano
+# clocks (V11, Y13, E11 - PL 50MHz)
+# clocks (E20, D20 - HPS 25MHz)
+
+set_location_assignment PIN_V11 -to sys_clk
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to sys_clk
+
+# Switches
+
+set_location_assignment PIN_Y24  -to gpio_bd_o[0]
+set_location_assignment PIN_W24  -to gpio_bd_o[1]
+set_location_assignment PIN_W21  -to gpio_bd_o[2]
+set_location_assignment PIN_W20  -to gpio_bd_o[3]
```

Comment:
The SWITCHES should be inputs for the FPGA

Reply:
Fixed it. Thank you.